gamemodes/terrortown/gamemode/server/sv_player.lua

```diff
@@ -656,10 +659,10 @@ local function CheckCreditAward(victim, attacker)
 				for k = 1, #plys do
 					local ply = plys[k]
 
-					if ply:IsActive() and ply:IsShopper() and ply:IsInTeam(attacker) and not ply:GetSubRoleData().preventKillCredits then
-						ply:AddCredits(amt)
+			if ply:IsActive() and ply:IsShopper() and ply:IsInTeam(attacker) and not ply:GetSubRoleData().preventKillCredits then
+				ply:AddCredits(amt)
 
-						--LANG.Msg(GetRoleTeamFilter(TEAM_TRAITOR, true), "credit_kill_all", {num = amt})
+				--LANG.Msg(GetRoleTeamFilter(TEAM_TRAITOR, true), "credit_kill_all", {num = amt})
```

Comment:
I think this line could be removed

Reply:
I don't think i changed this line and i don't understand why it's highlighted.

A lot of stuff in this function can be removed/rewritten and there are some features that might be worth implementing (allowing credit rewards for any combination of roles and teams)
I decided to only fix the bug (add the check) for now and give this another look at a later point.

Reply:
wait, nvm
my diff tool doesn't show whitespace changes